javascript/ecmascript-2015/function-improvements/method-shorthand.md

```diff
@@ -36,7 +36,7 @@ return {
 };
 ```
 
-With ES6 this can now become:
+Since ES6 this can now become:
```

Comment:
Actually, maybe remove the "since ES6" completely?

Reply:
removed

Reply:
done